src/github.com/getlantern/flashlight/genconfig/genconfig.go

```diff
@@ -69,6 +75,7 @@ func main() {
 	runtime.GOMAXPROCS(numcores)
 
 	loadDomains()
+	loadWhitelists()
```

Comment:
`proxiedsites` sounds great!  Yeah, `whitelist` is historical - that's what we've been using, but this is a good opportunity to get more precise.